parsec/core/backend_connection/proxy.py

```diff
@@ -70,6 +73,7 @@ def _get_proxy_from_pac(url: str, hostname: str) -> Optional[str]:
 
     try:
         proxies = pacfile.find_proxy_for_url(url, hostname)
+        assert proxies is not None, "Proxy list is None"
```

Comment:
Good catch! Maybe log the error and return `None` in this case, instead of failing the assertion